test/unified_format.py

```diff
@@ -874,6 +874,10 @@ def run_entity_operation(self, spec):
             if expect_error:
                 return self.process_error(exc, expect_error)
             raise
+        else:
+            if expect_error:
+                self.fail('Excepted error %s but "%s" succeeded: %s' % (
```

Comment:
Wow, I am surprised this wasn't caught by the tests for the unified format itself. Nice!